src/util.js

```diff
@@ -44,7 +44,7 @@ export const recursiveMerge = (
     // TODO(jlfwong): Handle malformed input where a and b are not the same
     // type.
 
-    if (typeof a !== 'object') {
+    if (typeof a !== 'object' || Array.isArray(b)) {
```

Comment:
I did think about merging arrays but in the use case where I was seeing the bug (with fontFamily) I don't think that you want to merge because it would cause an unexpected result. As an example, suppose you have a base style that has a fontFamily array and a second style that you want to merge that overrides the fontFamily:
```
const base = {...
  fontFamily: ['Arial', 'Helvetica', 'sans-serif'],
...}
const override = {...
  fontFamily: ['Times New Roman', 'serif'],
...}
```
The you'd expect the merged style to have a `fontFamily` value that matched `override` but if you merged you'd end up with:
```
fontFamily: ['Arial', 'Helvetica', 'sans-serif', 'Times New Roman', 'serif']
```
which would cause unexpected styling.